tendrl/node_agent/atoms/generic/command.py

```diff
@@ -0,0 +1,20 @@
+from tendrl.node_agent.ansible_runner.ansible_module_runner \
```

Comment:
Remove this file from the PR, since we just want to merge the framework bits and not the ansible task related bits